### PR TITLE
jrnl: add page

### PR DESCRIPTION
**Pull request by OrangeDugongo**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/jrnl.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# jrnl
+
+> A simple journal application for your command line.
+
+- Insert new entry with your editor:
```
Comment (sbrl) [inline pages/common/jrnl.md]: Maybe `Insert a new entry`... would read better?

File: pages/common/jrnl.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`jrnl`
+
+- Quickly insert new entry:
```
Comment (sbrl) [inline pages/common/jrnl.md]: Maybe ...`insert a new entry:` would read better?

File: pages/common/jrnl.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`jrnl {{timestamp}}: {{title}}. {{content}}`
+
+- View last ten entries:
```
Comment (sbrl) [inline pages/common/jrnl.md]: Maybe `View the last ten entries:` would read better?/

File: pages/common/jrnl.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`jrnl -n {{10}}`
+
+- View everything that happened from the start of last year to the start of last march:
+
+`jrnl -from {{"last year"}} -until {{march}}`
+
+- Edit all entries tagged with @texas and @history:
+
+`jrnl {{@texas}} -and {{@history}} --edit`
```
Comment (agnivade) [inline pages/common/jrnl.md]: I guess the tags are "texas", "history" and not "@texas", "@history". In that case, please move the @ out of the token. And also, we can remove the @ from the descriptions.

File: pages/common/jrnl.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+> A simple journal application for your command line.
+
+- Insert a new entry with your editor:
+
+`jrnl`
+
+- Quickly insert a new entry:
+
+`jrnl {{timestamp}}: {{title}}. {{content}}`
```
Comment (agnivade) [inline pages/common/jrnl.md]: "timestamp" is not immediately apparent as to the format of the timestamp to be used. I think it's better we use a concrete example for this.

File: pages/common/jrnl.md (hunk 4 above)
Comment (OrangeDugongo) [reply pages/common/jrnl.md]: Yes, you're right but `-and` accepts other filters and not only tags. I guess to remove the @ from the description but i left @ inside the token

Comment (agnivade) [reply pages/common/jrnl.md]: Cool, let's do that.

**Pull request merged**